axum/src/extract/multipart.rs

```diff
@@ -49,7 +49,7 @@ use std::{
 /// ```
 ///
 /// For security reasons it's recommended to combine this with
-/// [`ContentLengthLimit`](super::ContentLengthLimit) to limit the size of the request payload.
+/// [`DefaultBodyLimit`](super::DefaultBodyLimit) to limit the size of the request payload.
```

Comment:
Since `DefaultBodyLimit` doesn't (yet) apply to `Multipart` I think we need to mention https://docs.rs/tower-http/latest/tower_http/limit/index.html instead.